tests/h/formatters/annotation_moderation_test.py

```diff
@@ -72,16 +65,11 @@ def group(self, factories):
         return factories.Group()
 
     @pytest.fixture
-    def permission_granted(self, group):
-        has_permission = FakePermissionCheck()
-        has_permission.add_permission("moderate", group, True)
-        return has_permission
+    def has_permission(self):
+        def has_permission(permission, context):
+            """Return if we can do something in a context."""
 
-    @pytest.fixture
-    def permission_denied(self, group):
-        has_permission = FakePermissionCheck()
-        has_permission.add_permission("moderate", group, False)
-        return has_permission
+        return create_autospec(has_permission)
```

Comment:
I think you might be able to do:

```python
@pytest.fixture
def has_permission(self, pyramid_request):
    return create_autospec(pyramid_request.has_permission)
```

Although as mentioned in another PR I think using `pyramid.testing` rather than mocks is the gold standard here